docs/algorithms/preference_comparisons.rst

```diff
@@ -2,6 +2,94 @@
 Preference comparisons
 ======================
 
+The preference comparisons algorithms learns a reward function from the preferences between pairs of 
```

Comment:
I found this explanation a bit circular: "to predict the trajectory that will get a higher reward" --> aren't we learning the reward? It does make sense but you'd need to explain the difference between learned reward vs latent reward (if human preference comparisons) / ground truth reward (if synthetic).

How about something like:

```
The preference comparison algorithm learns a reward function from preferences between pairs of
trajectories. The comparisons are modeled as being generated from a Bradley-Terry (or Boltzmann rational) model, where the probability of preferring trajectory A over B is proportional to the exponential of the difference between the return of trajectory A minus B. In other words, the difference in returns forms a logit for a binary classification problem, and accordingly the reward function is trained using a cross-entropy loss to predict the preference comparison.